gallery/events.html

```diff
@@ -0,0 +1,257 @@
+                        <li><a href="https://drive.google.com/open?id=1lP76R5LvfFHVYH5yQlC2aZFOaIqu2yV9">Second Year</a></li>
+                        <li><a href="https://drive.google.com/open?id=1UqkfxmiAw5hJgbcZFW_mWRGHYW8uVp-j">Third Year</a></li>
+                      </ul>
+                    </li>
+                </li>
+                <li class="has-children"><a href="#">Past Pappers</a>
+                  <ul class="dropdown">
+                    <li><a href="https://drive.google.com/open?id=1xhNMyx-A7Twg9XdsF6EZsmX-H8KaJoGq">First Year</a></li>
+                    <li><a href="https://drive.google.com/open?id=1HOfMim4Fit6gRWh-8eEOXzNrHy097kTw">Second Year</a></li>
+                    <li><a href="https://drive.google.com/open?id=1ZPqZOYGo-uRwsqgiX_VldQqgwFHGu7iU">Third Year</a></li>
+                  </ul>
+                </li>
+                </li>
+              </ul>
+              </li>
+              </ul>
+            </nav>
+          </div>
+
+          <div class="col-6 col-xl-2 text-right" data-aos="fade-down">
+            <div class="d-none d-xl-inline-block">
+              <ul class="site-menu js-clone-nav ml-auto list-unstyled d-flex text-right mb-0" data-class="social">
+                <li>
+                  <a href="https://www.facebook.com/spec.ece/" class="pl-0 pr-3"><span class="icon-facebook"></span></a>
+                </li>
+                <li>
+                  <a href="https://twitter.com/SPEC__NITH" class="pl-3 pr-3"><span class="icon-twitter"></span></a>
+                </li>
+                <li>
+                  <a href="https://www.instagram.com/s.p.e.c_nith/" class="pl-3 pr-3"><span class="icon-instagram"></span></a>
+                </li>
+                <li>
+                  <a href="https://in.linkedin.com/in/s-p-e-c-nith-40214b197" class="pl-3 pr-3"><span class="icon-linkedin"></span></a>
+                </li>
+              </ul>
+            </div>
+
+            <div class="d-inline-block d-xl-none ml-md-0 mr-auto py-3" style="position: relative; top: 3px;"><a href="#" class="site-menu-toggle js-menu-toggle text-black"><span class="icon-menu h3"></span></a></div>
+
+          </div>
+
+        </div>
+      </div>
+
+    </header>
+
+
+
+    <div class="site-section" data-aos="fade">
+      <div class="container-fluid">
+
+        <div class="row justify-content-center">
+
+          <div class="col-md-7">
+            <div class="row mb-5">
+              <div class="col-12 ">
+                <h2 class="site-section-heading text-center">Events</h2>
+              </div>
+            </div>
+          </div>
+
+        </div>
+        <div class="row" id="lightgallery">
+          <div class="col-sm-6 col-md-4 col-lg-3 col-xl-2 item" data-aos="fade" data-src="bg-img/events/e1.jpg">
+            <a href="#"><img src="bg-img/events/e1.jpg" alt="IMage" class="img-fluid view"></a>
```

Comment:
@baamaboi remove anchor tag before image tag. I guess they are of no use. Check for a single image if there is any change in the carousel. If not, then do the same for the rest of the images.
And make sure to use the backslash in every path.

Reply:
The tag is used to change the cursor to a pointing hand. Same can be done via CSS. Please let me know which to use.

Reply:
Keep it as it is. But next time, style it with CSS.